.github/workflows/zxc-e2e-test.yaml

```diff
@@ -104,10 +148,11 @@ jobs:
       - name: Setup E2E Tests
         run: | 
           npm link
-          ./test/e2e/setup-e2e.sh
+          ${{ env.CG_EXEC }} ./test/e2e/setup-e2e.sh
```

Comment:
seems no need to add `env.` prefix
https://github.com/hashgraph/hedera-services/blob/e1c1665d23c75dadc70e0cc37d52d87067410b0e/.github/workflows/node-zxc-compile-application-code.yaml#L636

Reply:
I tried that first.  But, when I started deviating from the simple CG_EXEC command line that hedera-services had, I had issues.  This was a simple solution.

see how it starts adding quotes that I didn't have?
```
+ CGROUP_LOGLEVEL=DEBUG cgexec -g 'cpu,memory:user.slice/user-$(id' '-u).slice/user@$(id' '-u).service/e2e-9066903779' ./test/e2e/setup-e2e.sh
/runner/_work/_temp/7472a0f4-8494-4971-a8e0-625c16c37637.sh: line 4: CGROUP_LOGLEVEL=DEBUG: command not found
```
https://github.com/hashgraph/solo/actions/runs/9066903779/job/24910904625